### ISSUE TITLE
Discussion: Add source-level information to final- and secondary energy variables

### ISSUE THREAD

**Issue by danielhuppmann** [IAMconsortium/common-definitions#39]
Most final-energy variables (in ENGAGE, NAVIGATE and other projects) have a split by sector, secondary energy carrier, and primary energy from which that secondary energy carrier was produced, e.g., "Final Energy|Industry|Gases|Biomass“. However, in practice, the source of the secondary-energy carrier cannot be known exactly after the fuel is blended into the transmission system, e.g., electricity in the grid or methane molecules in a pipeline. Including all these combinations leads to an enormous number of variables that only offer spurious precision. Therefore, this sub-categorization was not carried over into this new variable list.

However, @Renato-Rodrigues objected to this exclusion in https://github.com/IAMconsortium/common-definitions/issues/34#issuecomment-1804150484.

This issue is aimed to discuss whether (and which) secondary- and final-energy variables should have information about the source of the energy source.

**Comment by robertpietzcker** [comment]
I think this information (are the liquids used in a specific sector sourced from fossil or biogenic origin) can be important for any study focusing on the EU or sub-levels, where you have different policies targeting eg industry and transport - thus the shares probably will be different across sectors. 

This can also happen with grid-based carriers like gases, hydrogen or heat, as you may have unconnected/partially connected grids. 

So I would make this split part of the variable tree, so that for projects where this information is needed it can easily be supplied and doesn't have to be redefined/discussed again, but to me the variables that detail this on sectoral level would clearly be variables of low priority that I usually would not request teams to report. (only the overall split - so FE|Liquids|Biomass, but not how this splits up into Transport/industry/buildings)

**Comment by orichters** [comment]
I agree that we should rather include these splits in the template and projects can decide not to report them, compared to having different projects reinvent wheels. Maybe we can add a "tier" marking stating which variables are crucial and which might be left out.
